addons/mail/static/src/discuss/core/common/notification_settings.xml

```diff
@@ -4,11 +4,11 @@
     <t t-name="discuss.NotificationSettings">
         <ActionPanel title.translate="Notification Settings" resizable="!!env.inMeetingView" initialWidth="400" icon="'fa fa-bell'">
             <div class="o-discuss-NotificationSettings">
-                <t t-if="props.thread.self_member_id?.mute_until_dt">
+                <t t-if="props.thread.channel?.self_member_id?.mute_until_dt">
```

Comment:
@tsm-odoo it's the same in the whole diff pretty much, I believe this PR is premature.

There are many more components/concept that should use channel model before moving the fields, as this will just require extra diff if we rename the fields first.

notification_settings should work with a channel and not a thread, there should be no guard